test_cases/TestCase_0.json

```diff
@@ -1,23 +1,23 @@
 {
     "id": "TestCase_0",
-    "name": "what treats MONDO:0001119",
-    "description": "Basis_treats_Premature_Menopause; Hexadrin_treats_Premature_Menopause; Busulfan_treats_Premature_Menopause; Lasofoxifene_treats_Premature_Menopause; Busulfan_treats_Premature_Menopause; estradiol_treats_Premature_Menopause; progestin_treats_Premature_Menopause",
+    "name": "what treats MONDO:0011426",
```

Comment:
Is there a way for Test Cases to persist? It seems like the Test Cases are getting overwritten each time, which just makes it hard to tell any diffs.

Reply:
it will (hopefully) once be easier when we have DH in place.  right now the spreadsheet is changing almost every time I parse it.